pages/admin/preview.js

```diff
@@ -1,76 +0,0 @@
-import styles from 'styles/pages/preview.module.scss';
-
-import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import bem from 'bem-css-modules';
-
-import Article from 'components/articles/Article';
-import ArticleCard from 'components/articles/cards/ArticleCard';
-import { DEFAULT_SIZES, SQUARE_SIZES } from 'components/articles/cards/CardWrapper';
-
-import { adminArticlesActions, adminArticlesSelectors } from 'redux/ducks/admin/articles';
-import { populateRequest } from 'utils/request';
-import { ArticleShape } from 'utils/customPropTypes';
-
-import useBoolean from 'hooks/useBoolean';
-
-const b = bem(styles);
-
-const mapStateToProps = (state, { lang }) => ({
-  article: adminArticlesSelectors.getPreview(state, lang),
-  error: adminArticlesSelectors.getError(state),
-});
-
-const ArticlePreview = ({ article, error }) => {
```

Comment:
I guess we should temporary disable preview-link in fibery then

btw what about preview mode? will it become a part of admin panel and live on admin subdomain?

Reply:
to be determined)